store/tikv/gcworker/gc_worker.go

```diff
@@ -515,63 +516,95 @@ func doGC(ctx goctx.Context, store tikv.Storage, safePoint uint64, identifier st
 	// Sleep to wait for all other tidb instances update their safepoint cache.
 	time.Sleep(gcSafePointCacheInterval)
 
+	log.Infof("[gc worker] %s start gc, safePoint: %v.", identifier, safePoint)
+	startTime := time.Now()
+	successRegions := 0
+	failedRegions := 0
+
+	ticker := time.NewTicker(gcJobLogTickInterval)
+	defer ticker.Stop()
+
 	req := &tikvrpc.Request{
 		Type: tikvrpc.CmdGC,
 		GC: &kvrpcpb.GCRequest{
 			SafePoint: safePoint,
 		},
 	}
-	bo := tikv.NewBackoffer(tikv.GcMaxBackoff, goctx.Background())
-
-	log.Infof("[gc worker] %s start gc, safePoint: %v.", identifier, safePoint)
-	startTime := time.Now()
-	regions := 0
 
+	bo := tikv.NewBackoffer(tikv.GcOneRegionMaxBackoff, ctx)
```

Comment:
Initialize it in following for loop seems better.

Reply:
It is hard to do because It should not be initialized when we meet region error.

Reply:
Got it.